packages/subgraph/config/network.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
seems like this file should stay for subgraph to build successfully

Reply:
but it's annoying that everytime I re-deploy a contract and update this file, it appeared to be in the changed files, and I remembered we already put `network.json` in the `.gitignore`, not sure why my git keeps on tracking this file :(

Reply:
add it to .gitignore and keep the file as is on the commited code?